api/v1alpha1/database_types.go

```diff
@@ -110,6 +110,57 @@ type Instance struct {
 	Type string `json:"type"`
 	// +optional
 	Profiles Profiles `json:"profiles"`
+	// +optional
+	// +kubebuilder:default:="{emptytminfo}"
```

Comment:
add a comment here as well about default:="{emptytminfo}"

Reply:
I added it, but then `make generate manifests` added the comment in the description of the CRD yaml, had to remove it ultimately. Nonetheless, I have provided information about this in the makefile comment and have added the relevant GitHub issue too.